ios/browser/svg/BUILD.gn

```diff
@@ -17,7 +17,6 @@ source_set("svg") {
   ]
 
   sources += [
-    "//third_party/skia/include/codec/SkPixmapUtils.h",
```

Comment:
this file doesn't exist in upstream GN files. We shouldn't add either to not create GN check failures in upstream `//skia` target.

Reply:
cc @Brandon-T